runner.rb

```diff
@@ -45,6 +45,23 @@ def show_entries
     puts "\nYou have logged #{@instahours.time_for_in_hours} hours so far for the week.\n\n"
   end
 
+  def get_date_from_array(entry_array)
+    begin
+      return Date.parse(entry_array[2])
+    rescue
+      return Date.today
+    end
+  end
+
+  def get_note_from_array(entry_array)
+    begin
+      Date.parse(entry_array[2]).strftime('%D')
```

Comment:
This is kind of hacky... using parse to force an error if the string can't be converted to a date
in that case,it's a note